src/inspector_agent.h

```diff
@@ -39,6 +39,7 @@ class InspectorSessionDelegate {
  public:
   virtual ~InspectorSessionDelegate() = default;
   virtual bool WaitForFrontendMessage() = 0;
+  // XXX SendMessageToFrontend() ?
```

Comment:
I mean that I find "OnMessage()" to not clearly indicate the purpose, from the name it could be used to send messages either way, whereas `SendMessageToFrontend()` clearly indicates the direction of the message flow.